Facebook/BusinessExtension/Observer/ViewContent.php

```diff
@@ -62,6 +61,7 @@ public function execute(Observer $observer) {
             'item_price' => $this->_magentoDataHelper->getValueForProduct($product)
           )
         ];
+        $customData['content_type'] = ($product->getTypeId() == \Magento\ConfigurableProduct\Model\Product\Type\Configurable::TYPE_CODE) ? 'product_group' : 'product';
       }
```

Comment:
for better readability, could you change it to `use \Magento\ConfigurableProduct\Model\Product\Type\Configurable `.

Reply:
Done